comparison.md

```diff
@@ -209,7 +209,7 @@ Help improve our documentation! If you find anything wrong or which could be imp
 				<td><strong>External Caching Systems (CDNs, Fastly)</strong></td>
 				<td>✅ Varnish automatically detects external Caching system and disables caching</td>
 				<td>✅</td>
-				<td>✅</td>
+				<td>✅ External caches respect Varnish/Drupal headers, Drupal modules available</td>
```

Comment:
Our Varnish installations work in a similar way, although they don't turn themselves off when CDNs are in place. So for those scenarios, we deliberately have double caching. The advantage here is simply that origin (Acquia) always stays fast, as long as the customer configured their site in that way.